internal/cli/auth/login.go

```diff
@@ -316,8 +421,10 @@ func LoginBuilder() *cobra.Command {
 	}
 
 	cmd.Flags().BoolVar(&opts.IsGov, "gov", false, "Log in to Atlas for Government.")
-	cmd.Flags().BoolVar(&opts.NoBrowser, "noBrowser", false, "Don't try to open a browser session.")
+	cmd.Flags().BoolVar(&opts.NoBrowser, "noBrowser", false, "Don't try to open a browser session to authenticate your User Account.")
```

Comment:
if this is false, will the CLI use only config files / env variables for auth? Or perhaps a session that is already authenticated? Wondering if we need to be clear about that behavior here.

I think we can remove the word "try" as it isn't clear to me if this is false if it just doesn't open a browser window or if it opens it if specific conditions are met or not met.
```suggestion
	cmd.Flags().BoolVar(&opts.NoBrowser, "noBrowser", false, "Don't open a browser session to authenticate your User Account.")
```